onnx/defs/rnn/defs.cc

```diff
@@ -6,6 +6,37 @@
 using AttrType = onnx::OpSchema::AttrType;
 using namespace onnx;
 
+std::function<void(OpSchema&)> RNNDocGenerator(const char* name) {
+    return [=](OpSchema& schema) {
+        schema.Attr("hidden_size", "Number of neurons in the hidden layer", AttrType::INT);
+        schema.Attr("direction", "Specify if the RNN is forward, reverse, or bidirectional. "
+                    "Must be one of forward (default), reverse, or bidirectional.",
+                    AttrType::STRING);
+        schema.Attr("clip", "Cell clip threshold. Clipping bounds the elements of a tensor "
+                    "in the range of [-threshold, +threshold] and is applied to the input "
+                    "of activations. No clip if not specified.",
+                    AttrType::FLOAT);
```

Comment:
Coming back to this: it feels a bit weird to have the default defined by the absence. And it is not consistent with other attributes that have defaults.

Reply:
Also, from the description it may not be clear where this is applied (I've seen people mistakenly clipping after the activation for instance, or only on parts of the equation). I'd add the clip in the equations.